js/pc.js

```diff
@@ -12,64 +12,6 @@ var PureClarity = {
 
         if (!this.config.enabled) return;
 
-        if (this.config.autocomplete.enabled){
-            $searchFields = jQuery(this.config.autocomplete.searchSelector);
-            if ($searchFields){
-                $searchFields.attr("id", "pc_search");
-            }
-        }
-
-        if (this.config.shop.do) {
-
-            var pcContainer = document.createElement('div');
-            var wrapper = document.createElement('div');
-            jQuery(pcContainer).addClass('shop');
-            jQuery(wrapper).addClass('pureclarity-wrapper');
-            jQuery(pcContainer).addClass('pureclarity-container').attr("data-pureclarity", "navigation_category:-1");
-            jQuery(this.config.shop.domSelector).wrap(wrapper).hide();
-            jQuery(".pureclarity-wrapper").append(pcContainer);
-            jQuery(wrapper).addClass('site-main');
-            if (this.config.shop.bmz1) {
```

Comment:
Do we no longer use this.config.shop.bmz1?

Reply:
yup, it was only used for inserting bmz content with injected search results from PC, search results bmz still work as intended without this